### PR TITLE
Can pass redirect url for after login

### PR DESCRIPTION
**Pull request by benvinegar**
/cc @getsentry/team @dcramer @lewisjellis

### CODE REVIEW

File: src/sentry/web/frontend/auth_login.py [hunk 1]
```diff
@@ -125,6 +125,11 @@ def handle_sso(self, request):
     @never_cache
     @transaction.atomic
     def handle(self, request):
+        redirect_next = request.GET.get('next', None)
```
Comment (dcramer) [inline src/sentry/web/frontend/auth_login.py]: this needs validation otherwise we'll get stupid security reports about phishing

**Review by dcramer — CHANGES_REQUESTED** [review]

File: src/sentry/web/frontend/auth_login.py (hunk 1 above)
Comment (benvinegar) [inline src/sentry/web/frontend/auth_login.py]: Take a look at the tests.

There's already validation in `sentry.utils.auth.get_login_redirect`.

**Review by dcramer — APPROVED** [review]
ok fine

**Comment by dcramer** [comment]
good job at tests

**Review by dcramer — APPROVED** [review]

**Comment by dcramer** [comment]
im merging this as i need it in another PR

**Pull request merged**